infra/base-images/base-runner-debug/Dockerfile

```diff
@@ -15,4 +15,8 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-runner
-RUN apt-get update && apt-get install -y gdb valgrind zip
+RUN apt-get update && apt-get install -y build-essential libgmp-dev valgrind zip
```

Comment:
Install the build deps and uninstall them in the same step where build GDB to avoid bloating the image.

Reply:
Done.